### ISSUE TITLE
The dimensionalities of encoded source embeddings

### ISSUE THREAD

**Issue by AaronWhite95** [CongBao/AutoencodedMetaEmbedding#3]
In model.py, class DAEME, build()
`self.encoders = [tf.layers.dense(ipt, dim, self.activ) for ipt, dim in zip(self.ipts, self.dims)]`
I think this will force the dimensionalities of source embedding and encoded source embedding to be the same, as in you paper, d1=d1', d2=d2'.
Is this necessary？

**Comment by CongBao** [comment]
@HWZX No, d1 and d1' (and/or d2 and d2') can be different. This version of code was frozen at the time the paper was published, and we did have unpublished further studies about the dimensionalities. Feel free to fork the repository and modify the code to meet your requirements. Hope this help.
